example/classic/example.nix

```diff
@@ -37,7 +37,10 @@
     providers = {
       dnsmasq = "dnsmasq-full";
     };
-    users.root.hashedPassword = "$6$n/dIMAV5QZyMp6UQ$fSvzsPZ8Vl1kzq9Mm3oQy81hxDkPqv04YPSlBOpqjMQKGu6xjcIuXrrfvf3Dcm8ea46oG8XtEPm6AViOFESF81";
+
+    #users.root.hashedPassword = "$6$n/dIMAV5QZyMp6UQ$fSvzsPZ8Vl1kzq9Mm3oQy81hxDkPqv04YPSlBOpqjMQKGu6xjcIuXrrfvf3Dcm8ea46oG8XtEPm6AViOFESF81";
+    users.root.hashedPasswordSecret = "root_pw";
```

Comment:
Can you add this secret to `secrets.yaml`? The secret key is listed in this file's header.

Reply:
Done